frontend/packages/dev-console/src/components/import/image-search/ImageStream.tsx

```diff
@@ -115,7 +115,7 @@ const ImageStream: React.FC = () => {
         {isNamespaceSelected && isImageStreamSelected && !isTagsAvailable && hasCreateAccess && (
           <div className="odc-imagestream-alert">
             <Alert variant="warning" title={t('devconsole~No Image streams tags found')} isInline>
-              {t('devconsole~No tags are available in image stream {{image}}', {
+              {t('devconsole~No tags are available in Image Stream {{image}}', {
```

Comment:
Just `Image` should be capitalized here so it should be `Image stream`.

Reply:
`Image Stream` is resource here so had it https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#image-stream LMK if if need to use `stream` instead of `Stream`

Reply:
Thanks for clarifying, `Stream` would be appropriate then.